packages/article-skeleton/src/native-ad.web.js

```diff
@@ -0,0 +1,36 @@
+const insertNativeAd = (children) => {
+	const child = children.find(item => item.name === 'paywall');
+	
+  if (!child) {
+    return children;
+  }
+
+	const clonedChildren = [...children];
+	const paywallChildren = child.children;
+	
+	let paraCount = 0;
```

Comment:
We tend to use `const` values instead of lets so that we aren't mutating the value and it's easier to read.

```suggestion
	const paragraphItems = paywallChildren.map((item, index) => ({...item, index})).filter(item => item.name === 'paywall')
	const indexToAdd = paragraphItems[6] ? item.index : null
```

Reply:
Array indexes start at 0, so the seventh paragraph element should be at position 6?

Reply:
have updated